### DISCUSSION TITLE
Module API Tutorial - outdated CLI instructions

### DISCUSSION THREAD

**Discussion by dcalkoj** [meshtastic/Discussions#78] [Firmware]
Currently working on creating my own module, trying for the past 2 days to work with the tutorial from the [documentation webpage.](https://meshtastic.org/docs/development/device/module-api/). In particular, I simply wanted to get ReplyModule working to see if I could successfully build compile/build onto my device.

From the "Getting Started" section, step 6 is no longer feasible. As far as I can tell, --sendping has been gone from the Meshtastic Python CLI for some time now? It didn't come to my attention that want_response would need to be set for replymodule to ever run. In fact, I'm still struggling a little bit to find a way to easily set the "want_response" bit other than modifying the firmware of another device.

![image](https://github.com/user-attachments/assets/cc33df97-7105-4812-95c9-db86f22408a2)

**Comment by garthvh** [comment]
What are you wanting to do, creating a whole module is not really necessary for most integrations. The reply module was deprecated and is no longer included in released builds.

**Reply by dcalkoj** [reply]
I'm working on writing a proof of concept for transmissions over a different communication frequency (currently small amplitude shift key rx/tx). 

Might be good to give that section of the documentation a note then if the tutorial is depreciated, haha
